assets/app/scripts/controllers/overview.js

```diff
@@ -12,9 +12,7 @@ angular.module('openshiftConsole')
     $scope.pods = {};
```

Comment:
I guess handling the empty case makes things more consistent one the one hand, but makes it hard if we want to be sure to handle all the solitary/ungrouped things first or last in a view loop